plugins/dev-create/bin/create.js

```diff
@@ -20,6 +20,7 @@ const path = require('path');
 const execSync = require('child_process').execSync;
 
 const root = process.cwd();
+const pluginDirName = 'plugins';
```

Comment:
Can we make this more generic. I don't think this one needs to be a requirement here. ie: I want it so that if "we, the Blockly team" use this, we would create our package under the plugin directory, but that can be a loose requirement, so others can just add it in their own repo. 

The logic I would have expected is, find where the root of the repo is, get the relative path between current directory and the root, and add the packageName to the end of that.